src/main/java/lotto/util/Split.java

```diff
@@ -0,0 +1,19 @@
+package lotto.util;
+
+import java.util.Arrays;
+
+public class Split {
+
+    private Split() {}
+
+    public static String[] from(String text, String regx) {
```

Comment:
불가피한 경우가 아니라면 컬렉션 객체의 형태로 항상 리턴하는 습관을 가지셨으면 좋겠습니다.
```suggestion
    public static List<String> from(...